index.html

```diff
@@ -27,6 +26,10 @@
         <img class="ball" src="images/cute me.jpg" />
         <p>Mackenzie Scott</p>
       </div>
+      <div class="person-frame-inner">
+        <img class="ball" src="/Users/dm/Documents/git-tutorial/images/drishti.JPG" />
```

Comment:
You need to add the path of the image that is now in our GitHub repository, because we don't have access to the desktop of your personal computer. Hint: `images/drishti.JPG`.